lib/v2/datasource/remote/model/transaction_model.dart

```diff
@@ -1,14 +1,19 @@
 import 'package:equatable/equatable.dart';
+import 'package:seeds/v2/datasource/remote/model/generic_transaction_model.dart';
+import 'package:seeds/v2/domain-shared/app_constants.dart';
+import 'package:seeds/v2/utils/read_times_tamp.dart';
+import 'package:seeds/v2/utils/string_extension.dart';
 
 class TransactionModel extends Equatable {
   final String from;
   final String to;
   final String quantity;
   final String memo;
-  final String timestamp;
-  final String transactionId;
+  final DateTime timestamp;
+  final String? transactionId;
```

Comment:
transaction ID is now optional since we are using this model for transactions we are about to send - which don't yet have a transaction ID

Before we were only using this to parse on chain transactions and show them in the transactions list.